packages/calypso-products/src/plans-list.tsx

```diff
@@ -3095,6 +3104,40 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getProductId: () => 2013,
 	},
 
+	[ PLAN_JETPACK_COMPLETE_BI_YEARLY ]: {
+		...getPlanJetpackCompleteDetails(),
+		...getAnnualTimeframe(),
+		getStoreSlug: () => PLAN_JETPACK_COMPLETE_BI_YEARLY,
+		getPathSlug: () => 'complete-bi-yearly',
+		getProductId: () => 2035,
+		getProductsIncluded: () => [
+			PRODUCT_JETPACK_BACKUP_T2_YEARLY,
+			PRODUCT_JETPACK_SCAN_BI_YEARLY,
+			PRODUCT_JETPACK_ANTI_SPAM_BI_YEARLY,
+			PRODUCT_JETPACK_VIDEOPRESS_BI_YEARLY,
+			PRODUCT_JETPACK_BOOST_BI_YEARLY,
+			PRODUCT_JETPACK_SOCIAL_ADVANCED_BI_YEARLY,
+			PRODUCT_JETPACK_SEARCH_BI_YEARLY,
+			PRODUCT_JETPACK_STATS_BI_YEARLY,
+			PRODUCT_JETPACK_CRM,
+		],
+		getWhatIsIncluded: () => [
+			translate( 'VaultPress Backup: Real-time backups as you edit' ),
+			translate( '1TB (1,000GB) of cloud storage' ),
+			translate( '1-year activity log archive' ),
+			translate( 'Unlimited one-click restores from the last 1 year' ),
```

Comment:
I think I already searched that before, but for the biennial plans, the activity log and restores only work up to 1 year, right?!

Reply:
I'm fairly sure that is correct, but that is a good question to confirm 🤔  Nice catch!

Reply:
It looks like that is also true for Jetpack Complete monthly, so it should be the same for Bi-Yearly since it is the same product